packages/editor-ui/src/components/ParameterInputFull.vue

```diff
@@ -165,18 +166,23 @@ export default mixins(
 				return this.ndvStore.inputPanelDisplayMode;
 			},
 			showMappingTooltip (): boolean {
-				return this.focused && this.isInputTypeString && !this.isInputDataEmpty && window.localStorage.getItem(LOCAL_STORAGE_MAPPING_FLAG) !== 'true';
+				return this.mappingTooltipEnabled && this.focused && this.isInputTypeString && !this.isInputDataEmpty && window.localStorage.getItem(LOCAL_STORAGE_MAPPING_FLAG) !== 'true';
 			},
 		},
 		methods: {
 			onFocus() {
 				this.focused = true;
+				// TODO: fix/patch Element UI tooltip: if 'manual' is set then 'open-delay' is ignored
```

Comment:
is this todo comment here needed? seems like you fixed it anyways here with a settimeout..

Reply:
yes, but that's a hack/workaround. I would be happy to use `open-delay` instead, it would be much clear

Reply:
removed